test/testenv/secretutil.go

```diff
@@ -159,3 +159,27 @@ func GetSecretDataMap(hecToken string, password string, pass4SymmKey string, idx
 	}
 	return updatedSecretData
 }
+
+// CheckSecretViaAPI return api response if it can be accessed
```

Comment:
The function is not performing a check. Maybe its better to call it `ExecuteCommandInPod` we can then use it in other places. 
We can return `(string, error)` and leave error handling part to caller.